src/botcmd.c

```diff
@@ -697,7 +696,7 @@ static void bot_nlinked(int idx, char *par)
     dprintf(idx, "error %s\n", BOT_YOUREALEAF);
   }
   if (s[0]) {
-    chatout("*** %s\n", s);
+    putlog(LOG_BOTS, "*", "%s.", s);
```

Comment:
We can, but I don't see any standard in keeping '***' in logmessages, especially with the time prefixed to it now.